lib/ext/heroku/command/addons.rb

```diff
@@ -439,9 +464,12 @@ def open
         end
       when 1 then
         addon_to_open = matches.first
-        launchy("Opening #{addon_to_open} for #{app}", app_addon_url(addon_to_open))
+        launchy("Opening #{addon_to_open["name"]} for #{app}", addon_to_open["web_url"])
```

Comment:
Given these add-ons on an app:

```sh-session
% h addons -a minasan
=== minasan Add-on Resources
heroku-postgresql:hobby-dev  @swimming-quietly-8857  DATABASE, HEROKU_POSTGRESQL_JADE
papertrail:choklad           @papertrail-8761924     PAPERTRAIL
```

This results in

```sh-session
% h addons:open papertrail -a minasan
Opening papertrail-8761924 for minasan... done

% h addons:open heroku-postgresql -a minasan
Opening swimming-quietly-8857 for minasan... done
```

Which is sort of interesting... could we use addon_to_open['addon_service']['name'] instead? That would be `Opening papertrail for minasan... done` and `Opening heroku-postgresql for minasan... done`, respectively, in the above example.